packages/common/src/interfaces/gridOption.interface.ts

```diff
@@ -345,6 +345,9 @@ export interface GridOption {
   /** Do we want to enable pagination? Currently only works with a Backend Service API */
   enablePagination?: boolean;
 
+  /** Odata case insensitive search */
+  caseInsensitiveOdataSearch?: boolean;
```

Comment:
I would rather have moved to the OData options, in the `OdataOption` interface here

https://github.com/ghiscoding/slickgrid-universal/blob/master/packages/odata/src/interfaces/odataOption.interface.ts

and since that interface is related to OData, we can rename the flag to simply be `caseInsensitive`